src/anyvar/extras/vcf.py

```diff
@@ -0,0 +1,80 @@
+"""Support processing and manipulation of VCF objects."""
+import logging
+from typing import Dict, List, Optional
+
+from ga4gh.vrs.extras.vcf_annotation import VCFAnnotator
+
+from anyvar.anyvar import AnyVar
+from anyvar.translate.translate import TranslatorConnectionException
+
+_logger = logging.getLogger(__name__)
+
+
+class VcfRegistrar(VCFAnnotator):
+    """Custom implementation of annotator class from VRS-Python. Rewrite some methods
+    and values in order to enable use of existing AnyVar translator.
+    """
+
+    def __init__(self, av: AnyVar):
+        """Initialize VCF processor.
+
+        :param av: complete AnyVar instance
+        """
+        self.av = av
+
+    def annotate(
+        self, vcf_in: str, vcf_out: Optional[str] = None,
+        vrs_pickle_out: Optional[str] = None, assembly: str = "GRCh38",
+    ) -> None:
+        """Annotates an input VCF file with VRS Allele IDs & creates a pickle file
+        containing the vrs object information.
+
+        :param vcf_in: The path for the input VCF file to annotate
+        :param Optional[str] vcf_out: The path for the output VCF file
+        :param Optional[str] vrs_pickle_out: The path for the output VCF pickle file
+        :param str assembly: The assembly used in `vcf_in` data
```

Comment:
I should make an issue in vrs-python for the paths to have type Path and not str
```suggestion
        :param vcf_out: The path for the output VCF file
        :param vrs_pickle_out: The path for the output VCF pickle file
        :param assembly: The assembly used in `vcf_in` data
```